pkg/util/validation/errors.go

```diff
@@ -48,7 +53,11 @@ type labelValueTooLongError struct {
 }
 
 func (e *labelValueTooLongError) Error() string {
-	return fmt.Sprintf("label value too long for metric: %.200q label value: %.200q", formatLabelSet(e.series), e.labelValue)
+	return globalerrors.FormatWithLimitConfig(
+		globalerrors.ErrIDSeriesLabelValueTooLong,
+		maxLabelValueLengthFlag,
+		"received series with label value length exceeding the limit, value: '%.200s' (truncated) series: '%.200s'",
```

Comment:
```suggestion
		"received series where the value length of its label exceeds the limit, value: '%.200s' (truncated) series: '%.200s'",
```

Reply:
I'm not fully convinced about the suggestion. So, a metric has 1+ label name-value pairs. We every every pair, and this error fires if one of such label **values** (only the value) are longer than the configured limit.

Reply:
Let's figure out the first one and propagate any language change throughout based on that.